packages/perseus/src/widgets/table/score-table.ts

```diff
@@ -10,29 +13,17 @@ import type {
 } from "../../validation.types";
 
 function scoreTable(
-    state: PerseusTableUserInput,
+    userInput: PerseusTableUserInput,
     rubric: PerseusTableRubric,
     strings: PerseusStrings,
 ): PerseusScore {
-    const filterNonEmpty = function (table: any) {
-        return _.filter(table, function (row) {
-            // Check if row has a cell that is nonempty
-            return _.some(row, _.identity);
-        });
-    };
-    const solution = filterNonEmpty(rubric.answers);
-    const supplied = filterNonEmpty(state);
-    const hasEmptyCell = _.some(supplied, function (row) {
-        return _.some(row, function (cell) {
-            return cell === "";
-        });
-    });
-    if (hasEmptyCell || !supplied.length) {
-        return {
-            type: "invalid",
-            message: null,
-        };
+    const validationResult = validateTable(userInput);
+    if (validationResult.type === "invalid") {
```

Comment:
Micro nit, feel free to ignore: we're going to be doing this a lot. Worth `isInvalid(validationResult)`? Or needless misdirection?

Reply:
I made a change based on your other comment about the return value for "success" (returning `null`). I think I like it that way.  I'll land this today but feel free to push back and we can easily try another pattern for returning success from these validators in a new PR. 

<img width="778" alt="image" src="https://github.com/user-attachments/assets/4cfe188f-e5cf-435b-a448-14236dcc4ac7">